infra/base-images/base-builder/compile

```diff
@@ -67,21 +67,6 @@ if [[ $FUZZING_ENGINE != "none" ]] && [[ $FUZZING_LANGUAGE != "jvm" ]]; then
   . compile_${FUZZING_ENGINE}
 fi
 
-if [[ $SANITIZER_FLAGS = *sanitize=memory* ]]
-then
-  # Take all libraries from lib/msan and MSAN_LIBS_PATH
-  # export CXXFLAGS_EXTRA="-L/usr/msan/lib $CXXFLAGS_EXTRA"
-  cp -R /usr/msan/lib/* /usr/lib/
```

Comment:
actually, /usr/msan/lib comes from llvm (and would contain libcxx?). we should probably keep this?

Reply:
Good point. I wonder if I falsely disabled MSAN on some projects because of this.